modules/primer-core/index.scss

```diff
@@ -22,6 +22,7 @@
 @import "primer-forms/index.scss";
 @import "primer-layout/index.scss";
 @import "primer-navigation/index.scss";
+@import "primer-pagination/index.scss";
```

Comment:
How does everyone feel about pagination in core vs product? I could go either way, but wanted to bring up the question. If it's not important for marketing then we can just keep it in product.

Reply:
Curious about this one too - wasn't sure where to put Box Overlay either 🤔